docs/Product/AboutTheOETProduct.mdx

```diff
@@ -6,25 +6,27 @@ sidebar_label: "About the OET Product"
 import MaintainerTag from "@site/src/components/MaintainerTag";
 import ControlledDocBanner from "@site/src/components/ControlledDocBanner";
 
-<MaintainerTag maintainerEmails={["jane.smith@example.com", "john.doe@example.com"]} />
+<MaintainerTag maintainerEmails={["mario.dzamarija@openenergytransition.org", "ekaterina.fedotova@openenergytransition.org"]} />
 
 # About the OET Product
-this should be same as sidebar_label and folder name
 
 <ControlledDocBanner />
 
-should be present on every controlled document.
+Open Energy Transition continues to build on open source software. PyPSA project, initially started in Academia, back in 2016, is being developed actively.
 
-## Sub Heading
+As of now, the PyPSA engine counts approx. 25,000 lines of Python code.
 
-At our company, we are committed to providing a safe and inclusive workplace for all employees. This policy outlines our expectations for behavior and conduct in the workplace.
+## Product Development
 
-## Another Sub Heading
+Open Energy Transition actively uses, and extend PyPSA, for the purposes of studies, consultations, workshops, presentations, conferences and projects.
```

Comment:
Thanks Mario.

extend --> extends

Reply:
Hey Fabrizio! Thanks a lot for your comments.

I have changed the 'extend' to 'extends'